documents/readthedoc/docs/source/Solutions/cross_language_framework_based_gramine/Readme.md

```diff
@@ -0,0 +1,54 @@
+Cross languages framework based on Gramine
+
+Intel SGX is a software guard extended instruction set designed to
+protect user privacy. Programs running in the SGX enclave will avoid
+external snooping and tampering. However, due to the diversity of user
+needs, their programs may be in different languages, such as Python,
+Java, and even Rust, Go. While the bottom layer of Intel SGX is
+developed based on C, there is no interface for other languages, and
+Gramine officially provided by Intel only supports limited languages,
```

Comment:
remove this line please, generally speaking, Gramine support all languages, but it only provides c language interface.